config/default/test/tasks/pregnancy_home_visit.spec.js

```diff
@@ -66,8 +59,11 @@ describe('pregnancy home visit tests', () => {
     await harness.setNow('1999-10-10');//10 weeks after LMP date
     const pregnancy = await harness.fillForm('pregnancy', ...pregnancyRegistrationScenarios.safe10Weeks);
     expect(pregnancy.errors).to.be.empty;
-    for (const day of getTaskTestDays(70, 32 * 7, pregnancyHomeVisitTaskFirst, TEST_INTERVAL_DAYS)
-      .concat(getTaskTestDays(32 * 7 + 1, MAX_DAYS_IN_PREGNANCY, pregnancyHomeVisitTaskSecond, TEST_INTERVAL_DAYS))) {
+
+    const daysToTest = getTaskTestDays(70, 32 * 7, pregnancyHomeVisitTaskFirst, TEST_INTERVAL_DAYS)
+      .concat(getTaskTestDays(32 * 7 + 1, MAX_DAYS_IN_PREGNANCY, pregnancyHomeVisitTaskSecond, TEST_INTERVAL_DAYS));
+    for (const day of daysToTest) {
```

Comment:
Nice, this is easier to read! `daysToTest` here corresponds to the days a CHW is supposed to visit a patient for ANC, right?

Reply:
looks like it takes _in_ the expected schedule and produces some [additional cases](https://github.com/medic/cht-core/blob/master/config/default/test/test-helpers.js#L35)? but yes - in spirit this test is validating the "schedule" for when the CHW should conduct a home pregnancy visit